### ISSUE TITLE
Incorrect valid range in WSUB_ExtData.yaml

### ISSUE THREAD

**Issue by bena-nasa** [GEOS-ESM/GEOSgcm_GridComp#849]
I realized the "valid range" in this file is wrong for the dataset:
https://github.com/GEOS-ESM/GEOSgcm_GridComp/blob/develop/GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSmoist_GridComp/WSUB_ExtData.yaml

To quote what is there now:
```
Collections:
  WSUB_SWclim_2005%m2.nc4:
    template: /discover/nobackup/dbarahon/DEV/SWclim/L72/SWclim_2005%m2.nc4
    valid_range: "2005-01-01/2005-12-31"
``1
That valid range is wrong, the dataset is only valid through 2005-12-01 so it should be changed to this:

```
Collections:
  WSUB_SWclim_2005%m2.nc4:
    template: /discover/nobackup/dbarahon/DEV/SWclim/L72/SWclim_2005%m2.nc4
    valid_range: "2005-01-01/2005-12-01"
```

**Comment by dbarahon** [comment]
Hi Ben,
That’s an easy fix. Should I update the tag?

Donifan

From: Ben Auer ***@***.***>
Reply-To: GEOS-ESM/GEOSgcm_GridComp ***@***.***>
Date: Thursday, October 26, 2023 at 2:34 PM
To: GEOS-ESM/GEOSgcm_GridComp ***@***.***>
Cc: "Barahona, Donifan (GSFC-6101)" ***@***.***>, Assign ***@***.***>
Subject: [EXTERNAL] [BULK] [GEOS-ESM/GEOSgcm_GridComp] Incorrect valid range in WSUB_ExtData.yaml (Issue #849)

CAUTION: This email originated from outside of NASA.  Please take care when clicking links or opening attachments.  Use the "Report Message" button to report suspicious messages to the NASA SOC.



I realized the "valid range" in this file is wrong for the dataset:
https://github.com/GEOS-ESM/GEOSgcm_GridComp/blob/develop/GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSmoist_GridComp/WSUB_ExtData.yaml

To quote what is there now:

Collections:

  WSUB_SWclim_2005%m2.nc4:

    template: /discover/nobackup/dbarahon/DEV/SWclim/L72/SWclim_2005%m2.nc4

    valid_range: "2005-01-01/2005-12-31"

``1

That valid range is wrong, the dataset is only valid through 2005-12-01 so it should be changed to this:



Collections:
WSUB_SWclim_2005%m2.nc4:
template: /discover/nobackup/dbarahon/DEV/SWclim/L72/SWclim_2005%m2.nc4
valid_range: "2005-01-01/2005-12-01"

**Comment by bena-nasa** [comment]
@dbarahon yes, please make a PR to fix this. It is simply the wrong valid range, please update to what I said
